### PR TITLE
[fix bug 1419246] Replace old Fx features pages with quantum equivalent.

### PR DESCRIPTION
**Pull request by jpetto**
## Description

Replaces old `/firefox/features/` pages with quantum equivalent. Deletes all unused files from old features pages.

## Issue / Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1419246

## Testing

Ensure all pages load/function as expected. Make sure CSS/images are loading, as paths have changed.

Make sure no orphaned files/assets were left behind.

Note that the diff can _mostly_ be ignored, as there was nothing new written in this PR. All deletions/additions noted are the result of overwriting `/firefox/features/{file}.html` with the contents of the pre-existing `/firefox/features/quantum/{file}.html` equivalent.

Integration tests [are passing](https://ci.us-west.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/130/pipeline).

### CODE REVIEW

**Comment by jpetto** [comment]
@peiying2 and/or @flodolo - I'm guessing the lang files will be automatically picked up based on the template names. Let me know if I'm wrong here.

**Comment by flodolo** [comment]
> Note that the diff can mostly be ignored, as there was nothing new written in this PR. All deletions/additions noted are the result of overwriting /firefox/features/{file}.html with the contents of the pre-existing /firefox/features/quantum/{file}.html equivalent.

I assume this PR doesn't have any actual impact on l10n, since the quantum page were using the old lang files?

**Comment by jpetto** [comment]
> I assume this PR doesn't have any actual impact on l10n, since the quantum page were using the old lang files?

I *think* this is true, but I added the L10N tag/pinged you to make sure. 😃

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -660,120 +660,6 @@ def ios_testflight(request):
...
-def features(request):
-    if switch('firefox-57-release'):
-        template = 'firefox/features/quantum/index.html'
-    else:
-        template = 'firefox/features/index.html'
-
-    return l10n_utils.render(request, template)
-
-
-class FeaturesPrivateBrowsingView(BlogPostsView):
```
Comment (alexgibson) [inline bedrock/firefox/views.py]: This is removing the blog posts from feature pages that are currently visible on prod. Is that intended?

Comment (jpetto) [reply bedrock/firefox/views.py]: Oh, nope. I got a little carried away cleaning up `views.py`. 😳 Restored views where needed.

**Pull request merged**